Mathlib/Algebra/Category/GroupCat/Basic.lean

```diff
@@ -134,7 +134,7 @@ set_option linter.uppercaseLean3 false in
 #align AddGroup.zero_apply AddGroupCat.zero_apply
 
 /-- Typecheck a `MonoidHom` as a morphism in `GroupCat`. -/
-@[to_additive]
+@[to_additive, nolint simpNF]
```

Comment:
This one is weird too; `nolint simpNF` is surely a no-op on defs?

Reply:
Yes. I was aspirationally hoping it got applied to the generated lemmas.